packages/connectors/src/metaMask.ts

```diff
@@ -55,6 +55,7 @@ export class MetaMaskConnector extends InjectedConnector {
           if (ethereum.isFordefi) return
           if (ethereum.isMathWallet) return
           if (ethereum.isOkxWallet || ethereum.isOKExWallet) return
+          if (ethereum.isOktoWallet) return
```

Comment:
Does Okto override MetaMask? If not, please remove this change.

Reply:
Hello @tmm, I have update the PR as suggested.